feature/feature_helper.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'open3'
+
+# Add the bin directory, to allow testing of gem executables as if the gem is
+# already installed.
+root_dir = RSpec::Core::RubyProject.root
+exec_dir = File.join(File::SEPARATOR, root_dir, 'bin')
+ENV['PATH'] = [exec_dir, ENV['PATH']].join(File::PATH_SEPARATOR)
+
+# Courtesy of:
+# https://raw.github.com/cupakromer/tao-of-tdd/master/adder/spec/support/
+#    capture_exec.rb
+def exec_io(*cmd)
```

Comment:
[reek] Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)